cli/settings.json

```diff
@@ -2,5 +2,6 @@
     "remote_hub": [
         "119.28.86.54:6616"
     ],
-    "initial_db_path": "../db/initial.trustnote.sqlite"
+    "initial_db_path": "../db/initial.trustnote-light.sqlite",
+    "db_path": "trustnote-light.sqlite"
```

Comment:
we only need the init_db_path
we should just hard code the database name in code, and remove this config item